Sprint-2/implement/cases.js

```diff
@@ -13,3 +13,9 @@
 
 // You will need to come up with an appropriate name for the function
 // Use the string documentation to help you find a solution
+
+function upperSnakeCase(str) {
+    return str.split(" ").join("_").toUpperCase();
+}
+console.log(upperSnakeCase("hello world"))
+console.log(upperSnakeCase("lord of the rings"))
```

Comment:
Again here the name should describe an action, for example "convertToSnakecase"

Reply:
name changed